src/main/java/com/walshydev/streamdeck4j/StreamDeck4J.java

```diff
@@ -399,8 +431,16 @@ public void setImage(@Nonnull String context, BufferedImage image, String type,
         setImage(context, Base64.getEncoder().encodeToString(imageBytes), type, destination);
     }
 
+    /**
+     * Sets the image of a specific button on the Stream Deck
```

Comment:
Probably worth mentioning that the base64 encoded string doesn't need to have the `data:image/{type};base64,` at the start as I will append it.